### PR TITLE
Add basic testing for HPC papi module

### PR DESCRIPTION
**Pull request by Avinesh**
Introducing a basic test for HPC papi module by executing a c test file based on papi library.

Related ticket: https://jira.suse.com/browse/PED-3032

Verification Run: https://openqa.suse.de/tests/10957829#

### CODE REVIEW

File: tests/hpc/mpi_master.pm [hunk 1]
```diff
@@ -122,7 +129,10 @@ sub run ($self) {
             assert_script_run($mpirun_s->slave_nodes("$exports_path{'bin'}/$mpi_bin"), timeout => 120);
             assert_script_run($mpirun_s->n_nodes("$exports_path{'bin'}/$mpi_bin", 2), timeout => 120);
         } else {
-            assert_script_run($mpirun_s->all_nodes("$exports_path{'bin'}/$mpi_bin"), timeout => 120);
+            # Skipping papi test on compute nodes for now
```
Comment (b10n1k) [inline tests/hpc/mpi_master.pm:132]: Can we just add that that this is likely due to the source file?

**Review by czerw — APPROVED** [review]

**Review by schlad — APPROVED** [review]

**Review by b10n1k — COMMENTED** [review]
Can you address my comment and move on with it?

**Review by b10n1k — CHANGES_REQUESTED** [review]
In additional some requests regarding the commit
- start with Capital
s/add basic testing for HPC papi module/Add basic testing for HPC papi module/
- Add description.

File: tests/hpc/mpi_master.pm (hunk 1 above)
Comment (Avinesh) [reply tests/hpc/mpi_master.pm:132]: Issue is with the loading of `papi` module itself  with current flow of execution in compute nodes, I don't think it is due to the source file.

**Pull request merged**